Explorer/Assets/DCL/Profiles/Self/SelfProfile.cs

```diff
@@ -106,6 +107,30 @@ IReadOnlyList<string> forceRender
             return await profileRepository.GetAsync(newProfile.UserId, newProfile.Version, ct);
         }
 
+        public async UniTask<Profile?> ForcePublishWithoutModificationsAsync(CancellationToken ct)
+        {
+            Profile? profile = await ProfileAsync(ct);
+
+            if (web3IdentityCache.Identity == null)
+                throw new Web3IdentityMissingException("Web3 Identity is not initialized");
+
+            if (profile == null)
```

Comment:
I think this path is redundant and should be asserted, I honestly believe it should not exist in another method either as it's an assertive signal that something went terribly wrong so we can simply throw an exception instead

Reply:
Done! https://github.com/decentraland/unity-explorer/pull/1601/commits/bffdd8ca2b2f8981f6faadc9c311041f9028d6cb